tests/js/client/aql/aql-mdi.js

```diff
@@ -309,19 +383,139 @@ function optimizerRuleMdi2dIndexTestSuite() {
     testCreateAsZkd: function () {
       let col = db._create(colName + "5");
       const idx = col.ensureIndex({
-        type: 'zkd',
+        type: "zkd",
         name: "myIndex",
-        fields: ['x', 'y'],
-        fieldValueTypes: 'double',
+        fields: ["x", "y"],
+        fieldValueTypes: "double",
       });
-      assertEqual('zkd', idx.type);
+      assertEqual("zkd", idx.type);
 
       const idx2 = col.index("myIndex");
       assertEqual(idx2.id, idx.id);
-      assertEqual(idx2.type, 'zkd');
+      assertEqual(idx2.type, "zkd");
 
       col.drop();
-    }
+    },
+
+    testInRangeNonStrict: function () {
+      const queries = [
+        aql`FOR d IN ${col} FILTER 0 < d.x && d.x < 1 RETURN d.x`,
+        aql`FOR d IN ${col} FILTER IN_RANGE(d.x, 0, 1, false, false) RETURN d.x`,
+      ];
```

Comment:
the test ist called `NonStrict` but `<` is the strict operator.

Reply:
You are right I messed up the meaning of strict